hs_core/management/commands/fix_resource_versioning.py

```diff
@@ -0,0 +1,117 @@
+import csv
+from django.core.management.base import BaseCommand
+from hs_core.hydroshare.utils import get_resource_by_shortkey, resource_modified
+
+
+class Command(BaseCommand):
+    help = "Fix versioning chain violation resources captured in an csv input file with three columns: " \
+           "isVersionOf, Resource, and isReplacedby"
+
+    def add_arguments(self, parser):
+        # csv filename with full path to load versioning chain violation resources from
+        parser.add_argument('input_file', help='input csv file name with full path to be processed and '
+                                               'load versioning chain violation resources from')
+
+    def handle(self, *args, **options):
+        input_file = options['input_file']
+        with open(input_file) as f:
+            reader = csv.reader(f)
+            headers = next(reader)
+            if len(headers) < 3 or headers[0].lower() != 'isversionof' or headers[1].lower() != 'resource' \
+                    or headers[2].lower() != 'isreplacedby':
+                print('headers are not correct, exit', flush=True)
+                return
+            count = 0
+            for row in reader:
+                if not row[1]:
+                    print('Resource column in the csv file cannot be empty, exit', flush=True)
+                    return
+                fixed = False
+                res_id = row[1]
+                try:
+                    res = get_resource_by_shortkey(res_id)
+                except Exception:
+                    print(f'Resource {res_id} does not exist', flush=True)
+                    continue
+
+                if not row[0] and not row[2]:
+                    # both isVersionOf and isReplacedby are empty, so clean up any versioning
+                    # relationships for this resource if any
+                    if res.metadata.relations.all().filter(type='isVersionOf').exists():
+                        res.metadata.relations.all().filter(type='isVersionOf').all().delete()
```

Comment:
simpler version: `res.metadata.relations.filter(type='isVersionOf').delete()`

Reply:
@pkdash What if ```res.metadata.relations.filter(type='isVersionOf')``` is None? That would raise an exception with the simpler version statement, right? Checking whether the relation exists first will avoid this issue, right?

Reply:
@hyi Even if a record doesn't exist it won't raise exception.